src/Explorer/Controls/Dialog.tsx

```diff
@@ -191,6 +197,7 @@ export const Dialog: FC = () => {
           {linkProps.linkText} <FontIcon iconName="NavigateExternalInline" />
         </Link>
       )}
+      {contentHtml && <Text>{contentHtml}</Text>}
```

Comment:
Will this only be text? shouldn't it be enough if you render the contentHtml as such? If needed move your <Text>...</Text> to where you define it